racket/collects/racket/HISTORY.txt

```diff
@@ -12,6 +12,7 @@ Change `unzip` to report errors for ill-formed archives, instead
  of silently ignoring certain errors
 Add `error-syntax->string-handler`
 Places: use `current-directory` value in created place
+Change `set/c` to support random generation
```

Comment:
This should be in 8.4, I think? 8.3 was already released.

Reply:
Yeah, it would need to be a new entry; here's [the plan](https://github.com/racket/racket/wiki/Release-Calendar), with dates.

Reply:
Ooops, sorry. Should be good now.